connexion/apis/flask_api.py

```diff
@@ -114,6 +116,7 @@ def _handlers(self):
         return self._internal_handlers
 
     @classmethod
+    @TracerDecorator
```

Comment:
is this specific to flask or will it work in general for other implementations too?

Reply:
It should work for other implementations aswell, because the tracer decorator doesn't take any flask specific parameter, but i didn't test it.

Reply:
In that case, can we add it also to the other api implementations? aiohttp, etc?

Would be nice to know it works universally and not just for flask :)

Reply:
Done. :) It is now available for all implementations. I made a quick test for aiohttp and it works as expected.